test/end2end.py

```diff
@@ -322,3 +331,27 @@ def test_unset_nodefile(self):
         self.assertEqual(schedm.what_sched(False)[0], Local)
         # restore env
         os.environ['PBS_NODEFILE'] = nodefile
+
+
+    def test_launcher_opt(self):
+        """Test --launcher command line option"""
+        install_fake_mpirun('mpirun', self.tmpdir)
+
+        # default behavior
+        ec, out = run_simple("%s %s --setmpi ihmpirun hostname" % (sys.executable, self.mympiscript))
+        regex = r'-bootstrap pbsdsh'
+        self.assertTrue(regex.find(out), "-bootstrap option is not pbsdsh (default)")
```

Comment:
include `out` in the error message so we can see what is there in case the check fails